### PR TITLE
Set dell as default node.

### PR DESCRIPTION
**Pull request by JuneZhou19**
@fub2 @turtle-fly @chenge3 @XiaowenJiang @xiar @WinnieXU1

### CODE REVIEW

File: test/functional/test_ipmi_console.py [hunk 1]
```diff
@@ -76,17 +76,20 @@ def setUp(self):
...
         os.system("pkill ipmi")
         os.system("pkill qemu")
+        os.system("pkill racadmsim")
 
     def test_start_stop_default_ipmi_console(self):
         self.node_name = "default"
         self.node_workspace = os.path.join(config.infrasim_home, self.node_name)
         os.system("infrasim node start")
         os.system("ipmi-console start")
+        time.sleep(20)
```
Comment (fub2) [inline test/functional/test_ipmi_console.py]: :+1:

**Comment by fub2** [comment]
+1

File: infrasim/model.py [hunk 2]
```diff
@@ -2141,6 +2145,10 @@ def init(self):
             racadm_obj.set_task_name("{}-racadm".format(self.__node_name))
             racadm_obj.set_log_path("/var/log/infrasim/{}/racadm.log".
                                     format(self.__node_name))
+
+            if "racadm_port" in self.__node:
```
Comment (turtle-fly) [inline infrasim/model.py]: @JuneZhou19 , "racadm_port" is deprecated, now we are using "port" under "racadm".

Please fix this part, and below 5 uses in test.

File: test/functional/test_ipmi_console.py [hunk 3]
```diff
@@ -184,13 +191,21 @@ def setUpClass(cls):
         # Wait SSH server coming up
         # FIXME: Need a good way to check if SSH server is listening
         # on port 9300
-        time.sleep(20)
         cls.ssh.set_missing_host_key_policy(paramiko.AutoAddPolicy())
-        cls.ssh.connect('127.0.0.1', username='', password='', port=9300)
+        #Retry SSH to make sure ipmi console started
+        for i in range(6):
+           try:
```
Comment (turtle-fly) [inline test/functional/test_ipmi_console.py]: Seems there's tab or un-aligned spaces here...

File: test/functional/test_ipmi_console.py [hunk 4]
```diff
@@ -184,13 +191,21 @@ def setUpClass(cls):
...
+               time.sleep(10)
+               cls.ssh.connect('127.0.0.1', username='', password='', port=9300)
+           except paramiko.ssh_exception.NoValidConnectionsError, e:
```
Comment (turtle-fly) [inline test/functional/test_ipmi_console.py]: This is much friendly to wait ipmi-console to be ready :1st_place_medal:

File: test/functional/test_node_ports.py [hunk 5]
```diff
@@ -111,16 +112,24 @@ def test_start_node1_then_start_node2_no_conflict_ports(self):
...
+        if 'dell' in self.node_info['type']:
+            assert "test1" in racadm_result
 
         self.node_info_2['name'] = "test2"
         # modify node configuration to resolve port conflict
         self.node_info_2['ipmi_console_ssh'] = 9301
         self.node_info_2['ipmi_console_port'] = 9001
         self.node_info_2['bmc_connection_port'] = 9101
+        if 'dell' in self.node_info_2['type']:
+            self.node_info_2['racadm_port'] = 10023
```
Comment (turtle-fly) [inline test/functional/test_node_ports.py]: As comment above, the code here should be:

    self.node_info_2["racadm"] = {}
    self.node_info_2["racadm"]["port"] = 10023

Below 4 tests need updates as well.

**Comment by fub2** [comment]
Sorry, merged it before all comments came in. @JuneZhou19

**Comment by fub2** [comment]
And it also requires separate PR ... Sorry @JuneZhou19

**Comment by JuneZhou19** [comment]
New PR submitted already. Never mind. @fub2

**Pull request merged**